Authentication/Extensions.swift

```diff
@@ -16,11 +16,11 @@ internal extension String {
          not have the key in the framework one.
      */
     internal var frameworkLocalized: String {
-        let localized: (from: NSBundle) -> String = {
+        let localized: (_ from: Bundle) -> String = {
```

Comment:
We already have a function to localise in `Core`, but it takes a list of parameters, and if called with no parameters, it fills everything with zeroes. I'll leave this as it is here, because I tried using Core's function and it doesn't work properly. We should create another `localized()` function that doesn't take any arguments.